src/main/java/website/frontrow/util/Point.java

```diff
@@ -87,6 +87,26 @@ public Point divide(double divisor)
     }
 
     /**
+     * Subtract a point from the current point.
+     * @param b Point to subtract.
+     * @return Result of subtraction.
+     */
+    public Point subtract(Point b)
```

Comment:
It is just a (random) point, the point does not have any meaning in particular.
Renaming it to point is an option, but it should (due to the type and context) be clear enough.